src/client/components/BaselayerControl.vue

```diff
@@ -0,0 +1,57 @@
+<template>
+  <div
+    class="baselayer-control baselayer-thumbnail"
+    @click="switchBaseLayer"
+  >
+    <img
+      :src="thumbnail"
+      :alt="altText"
+      class="baselayer-thumbnail__image"
+    >
+  </div>
+</template>
+
+<script>
+export default {
+  props: {
+    layers: {
+      type: Array,
+      required: true
+    },
+    switchHandler: {
+      type: Function,
+      required: true
+    }
+  },
+  data() {
+    return {
+      nextLayerIndex: 1
+    }
+  },
+  computed: {
```

Comment:
Rather than having a separate prop for the `altText` and the `thumbnail` would it maybe make more sense to have a computed property `nextLayer` which simply has two properties of its own `{ title: "...", thumbnail: "..." }`. I think it would make the template more expressive:
```vue
<span class="a11y-only">Switch to</span>
<img
  :alt="nextLayer.title"
  :src="nextLayer.thumbnail"
/>
```

Reply:
good one